### PR TITLE
[ES] Update `Countries_that_participated_in_osu!_tournaments`

### PR DESCRIPTION
**Pull request by agatemosu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Tournaments/Countries_that_participated_in_osu!_tournaments/es.md [hunk 1]
```diff
@@ -1,300 +1,332 @@
...
+| ::{ flag=UY }:: Uruguay | ![][false] | ![][true] | ![][true] | ![][false] | ![][false] | ![][false] | ![][false] | ![][false] | ![][false] | ![][true] | ![][true] | ![][true] |
+| ::{ flag=VE }:: Venezuela | ![][false] | ![][false] | ![][false] | ![][true] | ![][false] | ![][false] | ![][false] | ![][false] | ![][false] | ![][false] | ![][false] | ![][false] |
+| ::{ flag=VN }:: Vietnam | ![][false] | ![][true] | ![][false] | ![][false] | ![][false] | ![][false] | ![][false] | ![][false] | ![][false] | ![][true] | ![][true] | ![][true] |
+| **Total** | 28 | 32 | 32 | 32 | 32 | 32 | 32 | 32 | 32 | 43 | 48 | 51 |
 
-### osu!mania
+### ![][osu!taiko] osu!taiko World Cup
 
-#### Posiciones por países
+| País | 2011 | 2012 | 3 | 2014 | 2015 | 2016 | 2017 | 2018 | 2019 | 2020 | 2021 | 2022 |
```
Comment (agatemosu) [inline wiki/Tournaments/Countries_that_participated_in_osu!_tournaments/es.md:162]:
```md
| País | 2011 | 2012 | 3 | 2014 | 2015 | 2016 | 2017 | 2018 | 2019 | 2020 | 2021 | 2022 |
```

File: wiki/Tournaments/Countries_that_participated_in_osu!_tournaments/es.md [hunk 2]
```diff
@@ -1,300 +1,332 @@
...
-| ::{ flag=SE }:: Suecia | ![][false] | ![][true] 3rd | ![][true] | ![][true] |
-| ::{ flag=TW }:: Taiwán | ![][true] 2nd ×2 | ![][true] | ![][true] | ![][true] |
-| ::{ flag=TH }:: Tailandia | ![][true] | ![][true] | ![][true] | ![][false] |
-| ::{ flag=UA }:: Ukraine | ![][true] ½ | ![][false] | ![][false] | ![][false] |
-| ::{ flag=GB }:: Reino Unido | ![][false] | ![][true] | ![][true] | ![][true] |
-| ::{ flag=US }:: Estados Unidos | ![][true] | ![][true] | ![][true] | ![][true] |
-| ::{ flag=UY }:: Uruguay | ![][false] | ![][true] | ![][false] | ![][false] |
-| ::{ flag=VE }:: Venezuela | ![][false] | ![][false] | ![][false] | ![][true] |
-| Total number of teams | 32 | 28 | 32 | 32 |
+| País | 1 | 2 | 3 | 2013 | 2014 | 2015 | 2016 | 2017 | 2018 | 2019 | 2020 | 2021 |
```
Comment (agatemosu) [inline wiki/Tournaments/Countries_that_participated_in_osu!_tournaments/es.md:100]:
```md
| País | 1 | 2 | 3 | 2013 | 2014 | 2015 | 2016 | 2017 | 2018 | 2019 | 2020 | 2021 |
```

**Review by agatemosu — COMMENTED** [review]
This is correct?

File: wiki/Tournaments/Countries_that_participated_in_osu!_tournaments/es.md (hunk 1 above)
Comment (agatemosu) [inline wiki/Tournaments/Countries_that_participated_in_osu!_tournaments/es.md:162]: 3 is correct?

**Review by TicClick — APPROVED** [review]

**Comment by TicClick** [comment]
> 3 is correct?
> 
> ```gfm
> | País | 2011 | 2012 | 3 | 2014 | 2015 | 2016 | 2017 | 2018 | 2019 | 2020 | 2021 | 2022 |
> ```

I think it's an oversight -- must be `2013` as those are years

**Comment by TicClick** [comment]
I took a closer look and no, has to be `3`: https://github.com/ppy/osu-wiki/tree/master/wiki/Tournaments/TWC/3

**Review by TicClick — APPROVED** [review]

**Pull request merged**